packages/palette/src/elements/Button/Button.tsx

```diff
@@ -61,111 +61,6 @@ export interface ButtonBaseProps
   variantStyles?: any // FIXME
 }
 
-/**
- * Returns various colors for each state given a button variant
- * @param variant
- */
-export function getColorsForVariant(variant: ButtonVariant) {
-  const {
-    colors: { black100, black10, black30, white100, purple100 },
-  } = themeProps
-
-  switch (variant) {
-    case "primaryBlack":
-      return {
-        default: {
-          backgroundColor: black100,
-          borderColor: black100,
-          color: white100,
-        },
-        hover: {
-          backgroundColor: purple100,
-          borderColor: purple100,
-          color: white100,
-        },
-      }
-    case "primaryWhite":
-      return {
-        default: {
-          backgroundColor: white100,
-          borderColor: white100,
-          color: black100,
-        },
-        hover: {
-          backgroundColor: purple100,
-          borderColor: purple100,
-          color: white100,
-        },
-      }
-    case "secondaryGray":
-      return {
-        default: {
-          backgroundColor: black10,
-          borderColor: black10,
-          color: black100,
-        },
-        hover: {
-          backgroundColor: black30,
-          borderColor: black30,
-          color: black100,
-        },
-      }
-    case "secondaryOutline":
-      return {
-        default: {
-          backgroundColor: white100,
-          borderColor: black10,
-          color: black100,
-        },
-        hover: {
-          backgroundColor: white100,
-          borderColor: black100,
-          color: black100,
-        },
-      }
-    case "noOutline":
-      return {
-        default: {
-          backgroundColor: "rgba(0, 0, 0, 0)",
-          borderColor: "rgba(0, 0, 0, 0)",
-          color: black100,
-        },
-        hover: {
-          backgroundColor: white100,
-          borderColor: black100,
-          color: black100,
-        },
-      }
-    default:
-  }
-}
-
-/**
- * Returns css related to the passed in variant
- * @param variant
- */
-export const getStylesForVariant = (variant: ButtonVariant) => {
-  const { default: enabled, hover } = getColorsForVariant(variant)
-
-  return css`
-    ${() => {
-      return `
-          background-color: ${enabled.backgroundColor};
-          border-color: ${enabled.borderColor};
-          color: ${enabled.color};
-
-          @media ${themeProps.mediaQueries.hover} {
-            &:hover {
-              background-color: ${hover.backgroundColor};
-              border-color: ${hover.borderColor};
-              color: ${hover.color};
-            }
-          }
-        `
-    }};
-  `
-}
-
```

Comment:
This is interesting: I can't find anything that uses these functions in palette/force/volt. Seems safe to remove @damassi?

Reply:
I would double check the PR that this was introduced in to see if there's a paper trail, but yeah, I don't recall and this seems a bit unorthodox for whatever it may do 👍

Reply:
It *looks* like it was just put here by mistake when the React Native components were moved out.